tests/config/ray-cluster.mini.yaml.template

```diff
@@ -55,3 +55,64 @@ spec:
             name: dashboard
           - containerPort: 10001
             name: client
+  workerGroupSpecs:
```

Comment:
I would suggest to create a new templates other than reusing `cluster.mini.yaml` if this is still used by other tests?

Reply:
yeah, we can  change the name. This can be used by other tests too. But this is a template, that means this will be rendered to a ray cluster yaml. Itself is not a valid ray cluster yaml

Reply:
I am not sure whether some other tests want to use `minimum` cluster (1 node) template for testing

Reply:
We can make changes in the future if we need more worker nodes. The reason that I am not creating more than one head and one worker is that the VM resource is limited and too many nodes can slow down our tests and reduce its success rate.